protocol/v2/ssv/validator/committee.go

```diff
@@ -292,8 +298,13 @@ func (c *Committee) unsafePruneExpiredRunners(logger *zap.Logger, currentSlot ph
 	return nil
 }
 
-func (c *Committee) Stop() {
+func (c *Committee) Stopped() bool {
+	return c.stopped.Load()
+}
+
+func (c *Committee) stop() {
 	c.cancel()
+	c.stopped.Store(true)
```

Comment:
Maybe @iurii-ssv is right and its best to just check the context? I tend to agree.
@MatusKysel @anatolie-ssv @moshe-blox wdyt?

Reply:
+1